test/integration/targets/win_get_url/tasks/main.yml

```diff
@@ -27,6 +30,25 @@
     dependencies:
     - tcpip
 
+- name: gathering facts
+  setup:
+
+- block:
+  # unable install python package on Win2008 due to bug in win_chocolatey module (it wants interactive input)
+  - name: install python
+    win_chocolatey:
+      name: python
```

Comment:
Python seems to be a pretty big dependency to install here. Is there some sort of slim binary we can run as a service in the background. For example we currently test the FTP components by downloaded SlimFTPd binary https://github.com/ansible/ansible/blob/devel/test/integration/targets/win_get_url/tasks/main.yml#L12-L28. This is a tiny binary that we host in our S3 bucket to avoid network connectivity issues. The other trouble with using the python http.server is that we need to run it through async. Async seems to be slightly problematic in our CI environment and can cause the tests to be unstable from time to time.

On the other hand we can also test the checksum work by using an ftp or local file URI if the above becomes a bit too problematic.

Reply:
Hi @jborean93, we don't guarantee that replacing http test with ftp or even with local files is best approach, because it uses different webRequest methods, for example:
```
[System.Net.WebRequestMethods+Ftp]::DownloadFile
and
[System.Net.WebRequestMethods+Http]::Get
```
Could you kindly suggest any other way to test http(-s) test cases?

Reply:
@jborean93,

We checked some other options available but didn't manage to find better way of implementing http tests now. I agree that single slim binary would be better choice than Python. It may be [nginx](http://nginx.org/en/docs/windows.html) for example. But we're not sure which solution can be used for that case. I think we need a consolidated opinion from core team for that point. What do you think?

Reply:
One more choice is https://github.com/m3ng9i/ran as simple http server

Reply:
Fixed